charts/ssi-dim-wallet-stub/README.md

```diff
@@ -0,0 +1,76 @@
+## Helm char to deploy SSI DIM Wallet stub application
+
+### Source Code
+
+* <https://github.com/eclipse-tractusx/ssi-dim-wallet-stub>
+
+### Requirements
+
+| Repository | Name | Version |
+|------------|------|---------|
+| https://charts.bitnami.com/bitnami | keycloak | 22.1.0 |
+
+### Prerequisites
+- Kubernetes 1.19+
+- Helm 3.2.0+
+
+### Install
+```
+helm dep up charts/ssi-dim-wallet-stub
+kubectl create namespace wallet
+helm install wallet-stub -n wallet charts/ssi-dim-wallet-stub
+```
```

Comment:
Please also mention the install of the released chart, see example https://github.com/eclipse-tractusx/puris/blob/main/charts/puris/README.md#install

Reply:
Added with: https://github.com/eclipse-tractusx/ssi-dim-wallet-stub/pull/2/commits/64e4d7693197f3bb9a015787b9fbd46c47d7680c

Reply:
If you have added all information here, you could also remove the install.md on root level, or are you planning to fill it up with information?

Reply:
INSTALL.md is removed: https://github.com/eclipse-tractusx/ssi-dim-wallet-stub/pull/2/commits/1042e142d1c22db74eeaf95c02b4b0c50f62f329